DayCare/test_daycare.py

```diff
@@ -0,0 +1,130 @@
+import pytest
+from .models import daycare
+from django.core.exceptions import ValidationError
+from django.contrib.auth.models import User
+
+EMAIL = "test@gmail.com"
+USERNAME = "testuser01"
+PASSWORD = "pass"
+NAME = "Puppies"
+DESCRIPTION = "This is the first daycare test"
+PRICE_PER_DAY = 10
+CAPACITY = 50
+AREA = "Merkaz"
+CITY = "Tel-Aviv"
+ADDRESS = "The best street 5"
+
+
+@pytest.fixture
+def create_daycare_user():
+    return daycare.create(email=EMAIL, username=USERNAME, password=PASSWORD,
+                          name=NAME, description=DESCRIPTION,
+                          price_per_day=PRICE_PER_DAY,
+                          capacity=CAPACITY, area=AREA, city=CITY, address=ADDRESS)
+
+
+@pytest.mark.django_db()
+class TestDaycareModel:
+    def test_persist_daycare(self, create_daycare_user):
+        assert create_daycare_user.user in User.objects.all()
+        assert create_daycare_user in daycare.objects.all()
+
+    def test_del_daycare(self, create_daycare_user):
+        assert create_daycare_user.user in User.objects.all()
+        assert create_daycare_user in daycare.objects.all()
+
+        create_daycare_user.delete()
+        assert create_daycare_user not in daycare.objects.all()
+        assert create_daycare_user not in User.objects.all()
+
+    def test_del_user(self, create_daycare_user):
+        assert create_daycare_user in daycare.objects.all()
+        assert create_daycare_user in daycare.objects.all()
+
+        create_daycare_user.user.delete()
+        assert create_daycare_user not in daycare.objects.all()
```

Comment:
I am not sure why this test is needed, `test_del_daycare` already checks that User is deleted as well.

Reply:
because in the `test_del_daycare` i'm testing that if I delete the daycare it will delete the user also, here i am checking what happen if i delete only the user

Reply:
fixed the test